tests/fread/test-fread-large.py

```diff
@@ -106,6 +106,25 @@ def f(request):
 # Run the tests
 #-------------------------------------------------------------------------------
 
+
+# Make sure we can read/write csv and jay files larger than 4Gb on Windows
+# @pytest.mark.usefixtures("winonly", "is_release")
+@pytest.mark.usefixtures("winonly")
+def test_fread_4gb_plus(tempfile):
```

Comment:
There is also a `tempfile_jay` fixture. You can likewise make a `tempfile_csv` one.

Reply:
Nice to know, I guess it was needed since we detected jay files from their extensions?